libdnf/utils/utils.hpp

```diff
@@ -100,9 +105,37 @@ struct GoalJobSettings {
 
 private:
     friend class Goal;
-    bool get_strict(const libdnf::ConfigMain & cfg_main) const;
-    bool get_best(const libdnf::ConfigMain & cfg_main) const;
-    bool get_clean_requirements_on_remove(const libdnf::ConfigMain & cfg_main) const;
+    /// Resolve strict value and store used value
```

Comment:
store "used value" -> "used value" is the result of resolving, right?
Something like this:
Resolve the 'strict' value and save the result.